CHANGELOG.md

```diff
@@ -6,13 +6,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unrealeased]
 
+### BREAKING CHANGES
+
```

Comment:
We have noticed that the update of fastify-env may have broken the support to both `required` and `default` defined at the same time in the env jsonSchema

Reply:
let's add this detail to the changelog, also let's add the ajv changelog link so the user can see the breaking changes.